community/2024-10-30-community-meeting.mdx

```diff
@@ -18,41 +18,65 @@ title: 'Meeting Agenda - 2024-10-30'
 
 {/* Add summary of topics during the meeting. Each Topic should have an H3 heading. A list is usually sufficient for the recap */}
 
-### Topic 1
-
-- Note 1
-- Note 2
-
-### Topic 2
-
-- Note 1
-- Note 2
-
-## Weekly highlights
-
-### Issue of the Week
-
-- [title](https://github.com/wasmcloud/wasmcloud/issues/)
-  - summary
-
-### Documentation of the Week
-
-- [title](/docs/intro)
-  - summary
-
-## Community Updates
+### DEMO: [Full-stack Rust framework in the Leptos project](https://github.com/leptos-rs/leptos/pull/3063)
+
+- [Enzo Nocera](https://github.com/raskyld) is a Wasm enthusiast. He has experience working for many cloud providers but his work with wasmCloud is more of a personal passion.
+- Enzo has an exciting project which integrates Wasm (Wasmtime right now but later wasmCloud) with the Leptos project. The Rust ecosystem has been evolving in the background and so Enzo is demoing a full-stack framework to create backend and frontend applications with a single Rust codebase.
+- The aim was to prove we can use the same codebase to build a server side and browser side service based on modern Wasm standards.
+- 2 WASI contracts - filesyetem to serve static asets and WASI HTTP to stand up the server.
+- Super reative web app that runs entirely with Wasm as a demonstration of how quickly we can deploy web pages, and how we can build in backend integration.
+- This is one of the nicest experiences we've seen in building a front-end app with back-end integration. [Well worth checking out in the repo.
+- Still WIP but Enzo will share the details in Slack.
+- As always, it's really hard to express the joy of the demo - check out the recording below for Enzo's session, and ensuing community questions.
+
+### DEMO: wasmcloud_component HTTP wrappers
+
+- This is V2 of the components crate which incudes HTTP wrapper.
+- This was spurred by improved support for Wasm 32 and WASI P2 on a stable toolchain in Rust - this means it is a much nicer experience when starting and compiling with or without `wash-build`.
+- Brooks shows what the experience looks and feels like now the Rust toolchain is a little more mature.
+- This has now been published to our documentation as a template.
+- Now, rather than using wit-bindgen, we can use the wasmCloud component crate when we're writing in Rust.
```

Comment:
```suggestion
- Now, rather than using wit-bindgen, we can use the [`wasmcloud-component` crate](https://crates.io/crates/wasmcloud-component) when we're writing in Rust.
```